src/test/java/net/snowflake/ingest/TestUtils.java

```diff
@@ -20,7 +20,7 @@
 
 public class TestUtils
 {
-  //profile path
+  // profile path, follow readme for the format
   private final static String PROFILE_PATH = "profile.json";
```

Comment:
We can just refer to profile.json.example here directly, so whoever runs the test can modify the parameters in this file.

Reply:
I don't think we should do this. There is a file called ```profile.json.enc``` which is decrypted into ```profile.json```

The file ```.travis.yml```  decrypts the encrypted file using openssl to hide the parameters used to call an endpoint. If we modify this, it will not run the travis tests successfully.